package.json

```diff
@@ -22,9 +22,9 @@
     "lint:fix": "eslint . --cache --fix",
     "svelte:check": "svelte-check",
     "test:podlet-js": "vitest run --project podlet-js",
-    "test:backend":   "vitest run --project quadlet",
-    "test:frontend":  "vitest run --project frontend",
-    "test:shared":    "vitest run --project shared",
+    "test:backend": "vitest run --project quadlet",
```

Comment:
Is project really quadlet or should it ne backend ?

Reply:
vitest uses the field name in the package.json so here it is quadlet.

https://github.com/podman-desktop/extension-podman-quadlet/blob/653d8faac389d3869701ed472ed304edbebf8587/packages/backend/package.json#L2